legion/engine/llri-dx/detail/adapter.cpp

```diff
@@ -53,6 +53,40 @@ namespace llri
         return false;
     }
 
+    result Adapter::impl_querySurfacePresentSupportEXT(SurfaceEXT* surface, queue_type type, bool* support) const
+    {
+        switch(type)
+        {
+            case queue_type::Graphics:
+                *support = true;
+                break;
+            default:
+                *support = false;
+                break;
+        }
+        return result::Success;
+    }
```

Comment:
this looks pretty comical XD

Reply:
but i'm assuming it's cuz things are not as simple on the vk side

Reply:
yeah haha, DX12 has more statically defined requirements/limits in this case so all I can do is just return what I can find in the docs :P